.github/workflows/test-harness-acapy-credo-anoncreds.yml

```diff
@@ -0,0 +1,55 @@
+name: test-harness-acapy-credo-anoncreds
+# RUNSET_NAME: "ACA-PY to Credo Anoncreds"
```

Comment:
We’re trying to get consistent on using “AnonCreds” vs. “Anoncreds’.  Suggest that update be made.